website_product_brand/README.rst

```diff
@@ -0,0 +1,65 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Website Product Brand
+==================================
```

Comment:
This should have the same length of the title and put it also on top of the title to change style (see https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst)